docs/pages/install/docker.mdx

```diff
@@ -49,7 +49,7 @@ docker run -it -p 8444:8444 -p 8442:8442 --name=pelican-cache hub.opensciencegri
 Where most of the command overlaps with the one to run the Pelican origin server, with the following differences:
 * `-p 8444:8444 -p 8442:8442` publishes port `8444` and `8442` from the container to the same port on the host machine. Note that the admin website of the Pelican server runs on port `8444` by default, and the objects transfer endpoint of the Pelican cache server runs on port `8442` by default.
 
-> Note that Pelican docker image currently does not support binding a directory on your host machine as the directory for Pelican cache to store cache files, i.e. using `-v /host/machine:/run/pelican/cache/location`. We will have an update once we officialy support this flow.
+> **Note**: Pelican Docker image currently does not support binding a directory on your host machine as the directory for Pelican cache to store cache files, i.e. using `-v /host/machine:/run/pelican/cache/location`. We will have an update once we officialy support this flow.
```

Comment:
```suggestion
> **Note**: the Pelican Docker image currently does not support binding a directory on your host machine as the directory for a Pelican cache to store cached objects, i.e. using `-v /host/machine:/run/pelican/cache/location`. We will have an update once we officially support this flow.
```

And instead of "We will have an update once we officially support this flow." Maybe say something like this:
"This feature is a work in progress and will be available in a future Pelican version"

This sounds good to me but maybe others have another suggestion as well.

Reply:
Following up on Joe's `e.g` versus `i.e.` comment, they mean two different things and aren't interchangeable so we can't pick one over the other. The way I remember which to use is that "i.e." is Latin for "id est" and means "that is" whereas "e.g." is Latin for "exempli gratia" and means "for example". The former is to clarify something when there's only one way it can be clarified, and the latter is for any time you can construct an arbitrary example. In this case I think "e.g." is correct, but things look correct in most of the other usage.

Reply:
Thanks Justin for the explanation!